tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -2517,24 +2516,26 @@ def test_messed_up_function_values(self):
         data = response.data["data"]
         assert len(data) == 0
 
-    def test_context_fields(self):
+    def test_context_fields_between_datasets(self):
         self.login_as(user=self.user)
         project = self.create_project()
-        data = load_data("android")
+        event_data = load_data("android")
         transaction_data = load_data("transaction")
-        data["spans"] = transaction_data["spans"]
-        data["contexts"]["trace"] = transaction_data["contexts"]["trace"]
-        data["type"] = "transaction"
-        data["transaction"] = "/failure_rate/1"
-        data["timestamp"] = iso_format(before_now(minutes=1))
-        data["start_timestamp"] = iso_format(before_now(minutes=1, seconds=5))
-        data["user"]["geo"] = {"country_code": "US", "region": "CA", "city": "San Francisco"}
-        data["contexts"]["http"] = {
+        event_data["spans"] = transaction_data["spans"]
+        event_data["contexts"]["trace"] = transaction_data["contexts"]["trace"]
+        event_data["type"] = "transaction"
+        event_data["transaction"] = "/failure_rate/1"
+        event_data["timestamp"] = iso_format(before_now(minutes=1))
+        event_data["start_timestamp"] = iso_format(before_now(minutes=1, seconds=5))
+        event_data["user"]["geo"] = {"country_code": "US", "region": "CA", "city": "San Francisco"}
+        event_data["contexts"]["http"] = {
```

Comment:
This context is a bit misleading as the request interface doesn't actually live at this path. This will pass tests and work, but events captured in the wild would not have this shape and wouldn't work with the `http.*`fields we currently expose.

Reply:
Presumably we'll have to rewrite this once the http stuff solidifies.